runtime/tests/jni/module.xml

```diff
@@ -1,389 +1,390 @@
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedShortArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedIntArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedLongArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedFloatArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedDoubleArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetObjectArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedByteArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedBooleanArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedCharArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedShortArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedIntArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedLongArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedFloatArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedDoubleArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetObjectArrayElement"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperToReflectedField"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperToReflectedMethod"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedArrayLength"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperGetPackedArrayRegion"/>
-	<export name="Java_com_ibm_j9_packed_util_NativeTest_helperSetPackedArrayRegion"/>
-	<export name="Java_com_ibm_j9_packed_util_PackedCastNatives_boguscast"/>
-	<export name="Java_com_ibm_j9_packed_util_PackedCastNatives_bytesAreZero"/>
-	<export name="Java_com_ibm_j9_packed_util_PackedCastNatives_zeroObject"/>
-</exports>
-
-	<artifact type="shared" name="j9ben" appendrelease="false">
-		<include-if condition="spec.flags.module_jnitest" />
-		<options>
-			<option name="requiresPrimitiveTable"/>
-			<option name="prototypeHeaderFileNames" data="j9protos.h jnitest_internal.h"/>
-		</options>
-		<phase>core j2se</phase>
-		<exports>
-			<group name="all"/>
-		</exports>
-		<includes>
-			<include path="j9include"/>
-			<include path="j9oti"/>
-		</includes>
-		<makefilestubs>
-			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=1"/>
-			<makefilestub data="UMA_IGNORE_CODECOV=1"/>
-			<makefilestub data="UMA_DISABLE_DDRGEN=1"/>
-		</makefilestubs>
-		<libraries>
-			<library name="j9thr"/>
-			<library name="j9util"/>
-			<library name="j9utilcore"/>
-			<library name="jvm"/>
-		</libraries>
-	</artifact>
-</module>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why does this show up as 100% changed?

Reply:
Those two files had the wrong line-ends and were caught by the automated check. The real changes are in the first commit.

Reply:
Visit https://github.com/eclipse/openj9/pull/1326/files?w=1 to see the changes, ignoring whitespaces.